### PR TITLE
Add release notes for GOV.UK Frontend v3.11.0

### LINKED ISSUE #2116: Create release notes for v3.11.0
## What
Write the release notes for GOV.UK Frontend v3.11.0

## Why
To help users understand what changes have been made in the 3.11.0 release.

## Who needs to be involved 
Tech Writer and whoever is doing the release.

## Done when
- [x] release notes have been drafted
- [ ] release notes have been reviewed by a developer
- [ ] have been 2i'ed by Tech Writer community

### PR DESCRIPTION
**Pull request by EoinShaughnessy**
Closes #2116

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,11 +4,35 @@
...
 ### Fixes
 
+## 3.11.0 (Feature release)
+
+### New features
+
+#### Allow users to accept or reject cookies with a cookie banner
+
+You can now [use cookie banners](https://design-system.service.gov.uk/components/cookie-banner/) to allow users to accept or reject cookies which are not essential for making your service work.
+This was added in pull request #?: Add cookie banner.
```
Comment (EoinShaughnessy) [inline CHANGELOG.md:14]: We need a link and PR number here.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,11 +4,35 @@
...
+
+#### Group buttons and links
+
+You can now use a button group when you place two or more buttons together.
+Any links you include within a button group now line up visually with the buttons.
+Read more about [button groups on the Design System’s button page](https://design-system.service.gov.uk/components/button/#grouping-buttons).
```
Comment (EoinShaughnessy) [inline CHANGELOG.md:20]: Do we need to check that the link works?

File: CHANGELOG.md (hunk 1 above)
Comment (36degrees) [reply CHANGELOG.md:14]: https://github.com/alphagov/govuk-frontend/pull/2131

File: CHANGELOG.md (hunk 2 above)
Comment (hannalaakso) [reply CHANGELOG.md:20]: Looks like it'll be fine, linked to in https://deploy-preview-1472--govuk-design-system-preview.netlify.app/components/button/#grouping-buttons

**Review by hannalaakso — APPROVED** [review]

**Pull request merged**